docs/decorator-reference.md

```diff
@@ -97,19 +97,19 @@ export class User {
 }
 ```
 
-`@Column` decorator accept several options you can use:
+`@Column` accept several options you can use:
 
 * `type: ColumnType` - Column type. One of the type listed [above](#column-types).
```

Comment:
This document does not contain a header called "column types". So this link goes nowhere

Reply:
I moved different sections in places so many times...